lib/browser/api/app.js

```diff
@@ -42,9 +42,12 @@ Object.assign(app, {
 
 const nativeFn = app.getAppMetrics
 app.getAppMetrics = () => {
-  deprecate.removeProperty(nativeFn, 'privateBytes')
-  deprecate.removeProperty(nativeFn, 'sharedBytes')
-  return nativeFn.call(app)
+  let metrics = nativeFn.call(app)
+  const {memory} = metrics[0]
```

Comment:
Shouldn't it be deprecated on all items in the array, instead of just the first one?

```
for (let metric of metrics) {
  deprecate.removeProperty(metric.memory, 'privateBytes')
  deprecate.removeProperty(metric.memory, 'sharedBytes')
}
```

Reply:
Ah let me revise this suggestion. The above was made because I thought we were injecting a getter s.t. the warning would be generated only when the client code tried to access the deprecated property. That's not what we're doing.

So, revised question: shouldn't we do that? That seems better than warning about a deprecated property even to users who don't access that property.